airbyte-integrations/connectors/source-pendo/source_pendo/manifest.yaml

```diff
@@ -0,0 +1,3133 @@
+                                      features:
+                                        type: boolean
+                                      maxLength:
+                                        type: integer
+                                      omitPages:
+                                        type: boolean
+                                      maxInterval:
+                                        type: integer
+                                      trackEvents:
+                                        type: boolean
+                                      collapseDups:
+                                        type: boolean
+                                      predecessors:
+                                        type: boolean
+                                      followAcrossSessions:
+                                        type: boolean
+                    merge:
+                      type: object
+                      properties:
+                        fields:
+                          type: array
+                          items:
+                            type: string
+                        mappings:
+                          type: object
+                          properties:
+                            daysActive:
+                              type: string
+                            usageTrending:
+                              type: string
+                            page_siNpocwkDxZff1kSW6OhZLd59pY_null:
+                              type: string
+                            feature_0oE6eBQqe7sHyv6_Llk7i_UYTjo_null:
+                              type: string
+                            feature_Mh9p_zO8lJYWsonM7xyOXvpqEpE_null:
+                              type: string
+                            guide_JD317zFyWlXpl1Zvm1m4kB2xojo_guideViews:
+                              type: string
+                            page_3qt_EmeqXPi3zFwHEtFlW6qBi98_page_eventCount:
+                              type: string
+                            feature_3qYAs4rcyvdjAiINSbfrRJzkNH4_feature_eventCount:
+                              type: string
+                            feature_D67Hv6a9WGfNDDfTMok1xluo9_c_feature_eventCount:
+                              type: string
+                            poll_czx5ep1jrtu_14nCfODWTVf1OGZXupmQTzBdbHU_pollResponseAllTime:
+                              type: string
+                            poll_82cfeahyjdj_d0fWyIuRnhHZn2YFVltKospMBbU_pollResponseWithinDateRange:
+                              type: string
+                            poll_czx5ep1jrtu_14nCfODWTVf1OGZXupmQTzBdbHU_pollResponseWithinDateRange:
+                              type: string
+                            poll_jq84am6dxpm_14nCfODWTVf1OGZXupmQTzBdbHU_pollResponseWithinDateRange:
+                              type: string
+                            poll_x97q4ruspri_d0fWyIuRnhHZn2YFVltKospMBbU_pollResponseWithinDateRange:
+                              type: string
```

Comment:
What do these values refer to? They seem random but they also appear elsewhere here so just trying to get an idea.